rsmtool/reader.py

```diff
@@ -22,6 +22,47 @@
 from rsmtool.container import DataContainer
 
 
+
+def read_jsonlines(filename, converters=None):
+    """ Read jsonlines from a file. 
+    Normalize nested jsons with up to one level of nesting
+
+    Parameters
+    ----------
+    filename: str
+        Name of file to read
+    converters : dict or None, optional
+        A dictionary specifying how the types of the columns
+        in the file should be converted. Specified in the same
+        format as for `pd.read_csv()`.
+
+    Returns
+    -------
+    df : pandas DataFrame
+         Data frame containing the data in the given file.
+    """
+
+    df = pd.read_json(filename,
+                      orient='records',
+                      lines=True,
+                      dtype=converters)
+        
+    # let's check if we have nested columns
+    nested_cols = [c for c in df.columns if isinstance(df[c][0],dict)]
+
+    if len(nested_cols) > 0:
+        dfs = []
+        for col in nested_cols:
+            df_nested_column = pd.io.json.json_normalize(df[col])
+            dfs.append(df_nested_column)
+            df.drop(col, axis=1, inplace=True)
+        df_new = pd.concat([df] + dfs, axis=1)
+    else:
+        df_new = df
+        
+    return df_new
+
+
```

Comment:
Here's a more readable (and probably a bit more efficient) way of doing this I think. It also addresses @jbiggsets's concerns I think. This is supposed to replace the bits after the `pd.read_json()` call:

```python
dfs = []   
for column in df:
    try:
        df_column = pd.io.json.json_normalize(df[column])
    except AttributeError:
        df_column = df[column].copy()

    dfs.append(df_column)

df = pd.concat(dfs, axis=1)
```